packages/algob/src/lib/tx.ts

```diff
@@ -266,20 +268,70 @@ export async function makeAndSignTx (
   return [txns, signedTxn];
 }
 
+/**
+ * Signs transaction object(s) and returns raw signed transaction
+ * Note: Sign transaction is used to sign single transaction and `signTransactions` takes
+ * SDK transaction object, signer and signs it.
+ * @param txnAndSign Transaction object(s) with signers
+ */
+export function signTransactions (
+  txnAndSign: wtypes.TransactionAndSign[]
+): Uint8Array | Uint8Array[] {
+  let txns: Transaction[] = [];
+  const signers: wtypes.Sign[] = [];
+  txnAndSign.forEach(function (value, idx) {
+    txns[idx] = value.transaction;
+    signers[idx] = value.sign;
+  });
+  txns = algosdk.assignGroupID(txns);
+  return txns.map((txn: Transaction, index: number) => {
+    return signTransaction(txn, signers[index]);
+  });
+}
+
 /**
  * Execute single transaction or group of transactions (atomic transaction)
+ * executes `ExecParams` or `Transaction` Object, SDK Transaction object passed to this function
+ * will be signed and sent to network. User can use SDK functions to create transactions.
+ * Note: If passing transaction object a signer/s must be provided.
  * @param deployer Deployer
- * @param execParams transaction parameters or atomic transaction parameters
+ * @param transactionParam transaction parameters or atomic transaction parameters
  * https://github.com/scale-it/algo-builder/blob/docs/docs/guide/execute-transaction.md
+ * or TransactionAndSign object(SDK transaction object and signer parameters)
  */
 export async function executeTransaction (
   deployer: Deployer,
-  execParams: wtypes.ExecParams | wtypes.ExecParams[]):
-  Promise<ConfirmedTxInfo> {
-  deployer.assertCPNotDeleted(execParams);
+  transactionParam: (wtypes.ExecParams | wtypes.TransactionAndSign)
```

Comment:
let's rename `transactionParam` to `transactions`

Reply:
ok